### DISCUSSION TITLE
".Dirty "has any effect?

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#8962] [Build and Sync] [answered]
The upgraded version is "daemon: 1.16.0+mainnet+git.01254ab32.dirty+api1.5.0 Local: lotus version 1.16.0+mainnet+git. 01254ab32.dirty ",".Dirty "has any effect?

**Comment by rjan90** [comment, marked as answer]
Hey @ChaiWenyu1983! `.dirty` mean that a source file was modified. It is recommended to have a clean version, so if you follow these upgrade steps: https://lotus.filecoin.io/storage-providers/operate/upgrades/#upgrade-in-place, you should get a clean version 🙂
